apps/system/lockscreen/js/lockscreen.js

```diff
@@ -560,6 +568,7 @@
       this._lastUnlockedInterval = now - this._lastUnlockedTimeStamp;
       this._lastLockedTimeStamp = now;
 
+      this.playUnlockSound('/resources/sounds/lock.opus');
```

Comment:
Currently, There is no `details` object passed when the user clicks the `Lock Button`. Should I try to make it work in a similar way that `lockscreen_state_unlock.js` does it?